allspice/utils/list_components.py

```diff
@@ -175,14 +182,39 @@ def list_components_for_altium(
     schdoc_files_in_proj = _extract_schdoc_list_from_prjpcb(prjpcb_ini)
     allspice_client.logger.info("Found %d SchDoc files", len(schdoc_files_in_proj))
 
-    schdoc_jsons = {
-        schdoc_file: _fetch_generated_json(
-            repository,
-            _resolve_prjpcb_relative_path(schdoc_file, prjpcb_file),
-            ref,
-        )
-        for schdoc_file in schdoc_files_in_proj
-    }
+    tree = repository.get_tree(recursive=True)
+    files_in_this_repo = [file.path for file in tree]
+
+    schdoc_jsons = {}
+
+    for schdoc_file in schdoc_files_in_proj:
+        # For files not found in this repo, try the design reuse repos
+        if schdoc_file not in files_in_this_repo:
```

Comment:
`schdoc_file` here is a path relative to the path of the PrjPcb file, and `files_in_this_repo` has paths from the root of the repo. So if the project file is inside a folder this would fail for all sheets, even if they are in the repo. We'll need to resolve the relative path to absolute paths for this.